packages/client/hmi-client/src/services/model-configurations.ts

```diff
@@ -113,30 +113,31 @@ export function sanityCheck(config: ModelConfiguration): string[] {
 	return errors;
 }
 
-// cleans a model by removing distributions that are not needed
-export function cleanModel(model: Model): void {
-	const parameters: ModelParameter[] = getParameters(model);
-
-	parameters.forEach((p) => {
-		const max = parseFloat(p.distribution?.parameters.maximum);
-		const min = parseFloat(p.distribution?.parameters.minimum);
-
-		// we delete the distribution when there is partial/no distribution
-
-		if (Number.isNaN(max) || Number.isNaN(min)) {
-			delete p.distribution;
-		}
-	});
-}
-
 export function getInitial(config: ModelConfiguration, initialId: string): Initial | undefined {
 	return config.configuration.semantics?.ode.initials?.find(
 		(initial) => initial.target === initialId
 	);
 }
 
 export function getInitialSource(config: ModelConfiguration, initialId: string): string {
-	return config.configuration.metadata?.initials?.[initialId].source ?? '';
+	return config.configuration.metadata?.initials?.[initialId]?.source ?? '';
+}
+
+export function getInitialName(config: ModelConfiguration, initialId: string): string {
+	return config.configuration.metadata?.initials?.[initialId]?.name ?? '';
+}
+
+export function getInitialUnit(config: ModelConfiguration, initialId: string): string {
+	return config.configuration.metadata?.initials?.[initialId]?.unit ?? '';
+}
+
+export function getInitialDescription(config: ModelConfiguration, initialId: string): string {
+	return config.configuration.metadata?.initials?.[initialId]?.description ?? '';
+}
```

Comment:
Note name, unit, and description do not exist on initials in the AMR structure (neither does id), so all these fields will be blanks when you create the initial configuration out of an AMR. 

The initial data structure looks like this:
```
       {
         "target": "Susceptible",
         "expression": "0.999996300000000",
         "expression_mathml": "<cn>0.99999629999999995</cn>"
       }
```

Is this what we are expecting the user to do? Or were there some misconception in the design?

Reply:
All of these would be available to edit in the model page according to the [figma](https://www.figma.com/design/UXZ50KQ81tdD3aj1XqVCqz/Terarium---March-%2F-April-%2F-May-2024?node-id=800-58667&t=IkwkkMdJk7yz9nH4-0) designs.

Reply:
That is not my point. 

Is the design assuming that they will be at least partially populated? Because they will all be blank - e.g. are the designs matching reality and technical limitations? Cause it is weird to have a page of all blank fields.

Reply:
Ill bring this up in the design channel and see what they say

Reply:
here's the [slack thread](https://unchartedsoftware.slack.com/archives/C064P3QGVJ6/p1716389204519159)

Reply:
To summarize, there will be a message suggesting that metadata is missing.  Ill do this in separate [ticket](https://github.com/DARPA-ASKEM/terarium/issues/3678) though